### DISCUSSION TITLE
Clarifying the use of mini.fuzzy

### DISCUSSION THREAD

**Discussion by AmerM137** [nvim-mini/mini.nvim#1417] [Q&A] [answered]
I've been using mini.pick and I am quite happy with it. What does mini.fuzzy add to the mix? Does it enhance the pickers in any way?

I'd love some clarification on what exactly mini.fuzzy is, given that I am new to the vim/neovim way of doing things.
Doesn't neovim have built-in fuzzy finding already?

Do I need both mini.pick and mini.fuzzy?
Can I use mini.fuzzy only? If yes, how is it useful by itself?

Thanks!

**Comment by echasnovski** [comment, marked as answer]
'mini.fuzzy' is a library that [contains some functions for fuzzy matching](https://github.com/echasnovski/mini.nvim/blob/4228f166ee9db3e910eea1915e2d3683028add47/doc/mini-fuzzy.txt#L9-L16) and a function with custom sorter for 'nvim-telescope/telescope.nvim' (which is visibly faster than its defaults).

The 'mini.fuzzy' module severely predates even the possibility of something like custom fuzzy picker (which is 'mini.pick') being part of 'mini.nvim'.

No, you don't need 'mini.fuzzy' to keep using 'mini.pick'.
